File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        valores_vistos = {}
        for index, value in enumerate(nums):
            search = target - value
            if value in valores_vistos:
                return (index, valores_vistos[value])
            else:
                valores_vistos[search] = index


        

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> valuesSeems = new HashMap<>();

        for (int i = 0; i < nums.length; i++) {
            int value = target - nums[i];
            if (valuesSeems.containsKey(value)) {
                return new int[] {valuesSeems.get(value), i};
            }
            else {
                valuesSeems.put(nums[i], i);
            }
        }

        return new int[] {};
    }
}